tensorflow-proto/tensorflow-proto.cabal

```diff
@@ -11,32 +11,54 @@ copyright:           Google Inc.
+                     , Proto.Tensorflow.Core.Example.Feature
+                     , Proto.Tensorflow.Core.Framework.AllocationDescription
+                     , Proto.Tensorflow.Core.Framework.AttrValue
+                     , Proto.Tensorflow.Core.Framework.CostGraph
+                     , Proto.Tensorflow.Core.Framework.DeviceAttributes
+                     , Proto.Tensorflow.Core.Framework.Function
                      , Proto.Tensorflow.Core.Framework.Graph
+                     , Proto.Tensorflow.Core.Framework.KernelDef
+                     , Proto.Tensorflow.Core.Framework.LogMemory
                      , Proto.Tensorflow.Core.Framework.NodeDef
                      , Proto.Tensorflow.Core.Framework.OpDef
                      , Proto.Tensorflow.Core.Framework.ResourceHandle
+                     , Proto.Tensorflow.Core.Framework.StepStats
                      , Proto.Tensorflow.Core.Framework.Summary
                      , Proto.Tensorflow.Core.Framework.Tensor
+                     , Proto.Tensorflow.Core.Framework.TensorDescription
                      , Proto.Tensorflow.Core.Framework.TensorShape
+                     , Proto.Tensorflow.Core.Framework.TensorSlice
                      , Proto.Tensorflow.Core.Framework.Types
-                     , Proto.Tensorflow.Core.Protobuf.Config
-                     , Proto.Tensorflow.Core.Util.Event
-  other-modules:       Proto.Tensorflow.Core.Framework.AllocationDescription
-                     , Proto.Tensorflow.Core.Framework.CostGraph
-                     , Proto.Tensorflow.Core.Framework.Function
-                     , Proto.Tensorflow.Core.Framework.StepStats
-                     , Proto.Tensorflow.Core.Framework.TensorDescription
+                     , Proto.Tensorflow.Core.Framework.Variable
                      , Proto.Tensorflow.Core.Framework.Versions
+                     , Proto.Tensorflow.Core.Lib.Core.ErrorCodes
+                     , Proto.Tensorflow.Core.Protobuf.Config
+                     , Proto.Tensorflow.Core.Protobuf.ControlFlow
                      , Proto.Tensorflow.Core.Protobuf.Debug
+                     , Proto.Tensorflow.Core.Protobuf.MetaGraph
+                     , Proto.Tensorflow.Core.Protobuf.NamedTensor
+                     , Proto.Tensorflow.Core.Protobuf.QueueRunner
+                     , Proto.Tensorflow.Core.Protobuf.TensorBundle
+                     , Proto.Tensorflow.Core.Protobuf.SavedModel
+                     , Proto.Tensorflow.Core.Protobuf.Saver
+                     , Proto.Tensorflow.Core.Protobuf.TensorflowServer
+                     , Proto.Tensorflow.Core.Util.Event
+                     , Proto.Tensorflow.Core.Util.MemmappedFileSystem
+                     , Proto.Tensorflow.Core.Util.SavedTensorSlice
+                     , Proto.Tensorflow.Core.Util.TestLog
   build-depends:  proto-lens == 0.2.*
                 , proto-lens-protoc == 0.2.*
+                , proto-lens-protobuf-types == 0.2.*
```

Comment:
Yep; they're released now.  Updated `stack.yaml` to use the new versions.

Sometime next week there will be a new `lts-8.*` release with these new `proto-lens-*` versions, which will let us simplify the stack file further (other than `proto-lens-protobuf-types`, which might need to wait for the next LTS version`).